src/containers/oracle.js

```diff
@@ -148,9 +150,14 @@ class OraclePage extends React.Component {
             configName = 'SETTING';
             break;
           default:
+            console.warn('Oracle exists but cant determine status. ');
+            oracle = undefined;
             break;
```

Comment:
Basically, if we can't identify an oracle's status with above rules, we are gonna show 404 in the center of page, to which I haven't added style yet.

Reply:
👍